src/components/account-tab.yml

```diff
@@ -10,3 +13,9 @@ updateSuccessMessage:
 passwordResetEmailSent:
   en: The password reset link has been sent to {email}
   de: Der Link zum Zurücksetzen des Kennworts wurde an {email} gesendet
+closeAccountDetails:
+  en: All documents, public rooms and publicly uploaded resources will remain on the website and your username will be displayed as the collaborator. All other data (personal information, private rooms and privately uploaded resources) will be removed. You will no longer be able to log into the platform.
+  de: All documents, public rooms and publicly uploaded resources will remain on the website and your username will be displayed as the collaborator. All other data (personal information, private rooms and privately uploaded resources) will be removed. You will no longer be able to log into the platform.
+closeAccount:
```

Comment:
Let's please phrase the English text together, if I missed something, and then translate it 🙏